### ISSUE TITLE
package.json license statement not SPDX compliant

### ISSUE THREAD

**Issue by robertpatrick** [digitalbazaar/forge#1002]
In package.json, the project lists its license as `(BSD-3-Clause OR GPL-2.0)` but according to the [SPDX license page](https://spdx.org/licenses/), there is no `GPL-2.0` license code.  It seems like it needs to be either `GPL-2.0-only` or `GPL-2.0-or-later`.

I assume it should be `GPL-2.0-only` but this should really be corrected.

**Comment by davidlehn** [comment]
It used to exist, but looks like they recently in 2018 "obsoleted" `GPL-2.0`.
https://github.com/spdx/license-list-XML/blob/main/src/GPL-2.0.xml.

I don't know what it should be.  Need to diff the license text with each of their versions and see which matches.

Is some tooling complaining?

**Comment by robertpatrick** [comment]
@davidlehn yes, we have tooling based on SPDX that processes and gives credit to third-party packages.  Our tool is complaining...

**Comment by robertpatrick** [comment]
According to https://spdx.org/licenses/GPL-2.0.html, it seems clear that the new code from `GPL-2.0` is `GPL-2.0-only`.
